db/schema.rb

```diff
@@ -51,7 +51,6 @@
     t.datetime "updated_at", null: false
     t.text "instance_data", comment: "Additional client information, e.g. instance identity document"
     t.string "cloud_provider"
-    t.boolean "proxy_byos", default: false
```

Comment:
why is this column gone in the master schema?

Reply:
This one buffles me. It was never on `hw_infos` and always in `systems`. 
```
gondor λ grep -r 'proxy_byos' db/*                                                                                                                                       
migrate/20211017185107_add_proxy_byos_to_systems.rb:    add_column :systems, :proxy_byos, :boolean
migrate/20211017185107_add_proxy_byos_to_systems.rb:    change_column_default :systems, :proxy_byos, false
migrate/20211017185107_add_proxy_byos_to_systems.rb:    System.update_all(proxy_byos: false)
migrate/20211017185107_add_proxy_byos_to_systems.rb:    remove_column :systems, :proxy_byos
schema.rb:    t.boolean "proxy_byos", default: false
```

I think this is a superfluous addition in the wrong place? https://github.com/SUSE/rmt/blame/master/db/schema.rb#L54

Engines always use `systems.proxy_byos`.

Reply:
Weird, it was added by me: 
![image](https://github.com/SUSE/rmt/assets/582520/93af5029-3a25-46f3-9569-02156ec8c2e7)
But I cannot find any migration that added it there. Maybe the migration got deleted? But in the end it doesn't matter, because we'll drop the whole hw_infos table in one of the next updates.